app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -4,122 +4,111 @@ input:-webkit-autofill{
   background-image: none !important;
 }
 
-
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
-    color : $text-dark-grey;
-  }
-
-  input {
-    &:invalid,
-    &:invalid:required,
-    &:invalid:required:focus {
-      -webkit-box-shadow : none;
-      -moz-box-shadow : none;
-      box-shadow : none;
-      box-shadow : none;
-
-      border : none;
-      color : $text-dark-grey;
-    }
+    color: $text-dark-grey;
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
   }
 
   fieldset {
-    @include border-radius(5px);
-
-    -webkit-box-shadow : inset 0 1px 1px rgba(0,0,0,0.2), 0 1px 2px rgba(255,255,255,0.7);
-
-    margin-bottom : 1em;
-
-    background-color : #fff;
-
-    border : 1px solid $border-dark-grey;
-
-    input[type=text],
-    input[type=email],
-    input[type=password] {
-      @include box-shadow(0,0,0,0);
-      @include border-radius(0);
-
-      background : transparent;
-
-      padding : 10px;
-
-      margin-bottom : 0;
-      border : none;
+    margin-bottom: 1em;
+    background-color: #fff;
+    position: relative; /* To correctly place the entypo icon */
```

Comment:
Aha, `fieldset` is also the relative container for `label` That it good, although I would very much like it done without them.